scripts/platforms/workday_apply_v2.py

```
@@ -0,0 +1,424 @@
+    return {
+        "first name": parts[0] if parts else "",
+        "last name": " ".join(parts[1:]) if len(parts) > 1 else "",
+        "full name": full_name,
+        "email": safe_get("email"),
+        "phone": safe_get("phone"),
+        "linkedin": safe_get("linkedin"),
+        "portfolio": safe_get("portfolio"),
+    }
+
+
+async def apply():
```

Ruff (0.15.12)

[warning] Too many branches (34 > 12)

(PLR0912)